### PR TITLE
fix: incorrect templateRef

### LINKED ISSUE #6767: Listing when creating an NFT on Basilisk won't let you press Create NFT
### What happened?

Please Check: 

https://github.com/kodadot/nft-gallery/assets/36627808/1037fcb9-0d1d-4be3-b514-dd52ac242544

i know i put 0 KSM when listing, but with any value the issue will occur

### PR DESCRIPTION
**Pull request by stephenjason89**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] closes #6767 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR&usdamount=100&donation=true)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 769abca</samp>

Refactored the code for opening the shopping cart modal from the navbar. Moved the logic and configuration from `ShoppingCartModalConfig.ts` and `ShoppingCartButton.vue` to `Navbar.vue` to improve code organization and performance.

### <samp>🤖 Generated by Copilot at 769abca</samp>

> _`ShoppingCartButton`_
> _Simpler, no more `emit` -_
> _Autumn leaves falling_

### CODE REVIEW

File: components/bsx/input/TokenBalanceInput.vue [hunk 1]
```diff
@@ -34,6 +34,7 @@ const unit = computed(() => asset.value.symbol)
 const decimals = computed(() => asset.value.decimals)
 
 function checkValidity() {
+  console.log('token balance validity')
```
Comment (roiLeo) [inline components/bsx/input/TokenBalanceInput.vue:37]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]
code lgtm, mint no longer blocked

**Review by vikiival — APPROVED** [review]
I trust @roiLeo

**Comment by vikiival** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 4.46 USD/DOT ~ 4.484 $DOT
🧗 [16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR](https://polkadot.subscan.io/account/16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR)
🔗 [0x8afabcc3017c00b6568d8694a79644362c7401c75e239b3170ceaaadf7b96dd7](https://polkadot.subscan.io/extrinsic/0x8afabcc3017c00b6568d8694a79644362c7401c75e239b3170ceaaadf7b96dd7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by stephenjason89** [comment]
> I trust @roiLeo

I'll earn your trust, somday😄

**Pull request merged**